src/verifiers/documentStatus/didSigned/didSignedDocumentStatus.ts

```diff
@@ -119,12 +140,19 @@ const verifyV2 = async (document: SignedWrappedDocument<v2.OpenAttestationDocume
         issuance,
```

Comment:
we should probably add the details about revocation here

Reply:
got it